### PR TITLE
SW-405-communicate-with-findologic-api

### PR DESCRIPTION
**Pull request by zaifastafa**
The build is failing as the findologic-api PR is not merged yet. It will pass once the PR is merged into ~1.2

### CODE REVIEW

File: FinSearchUnified/Bundle/SearchBundleFindologic/QueryBuilder/NewQueryBuilder.php [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+        $this->searchNavigationRequest->addAttribute($key, $value);
+    }
+
+    /**
+     * @param string $usergroup
+     */
+    public function addUserGroup($usergroup)
+    {
+        $hashedGroup = StaticHelper::calculateUsergroupHash($this->shopKey, $usergroup);
+        $this->searchNavigationRequest->addGroup($hashedGroup);
```
Comment (TheKeymaster) [inline FinSearchUnified/Bundle/SearchBundleFindologic/QueryBuilder/NewQueryBuilder.php]: This one should actually be `addUserGroup`. I think FINDOLOGIC-API does not yet support this parameter, so please add another PR that adds this parameter. Thanks.

Comment (TheKeymaster) [reply FinSearchUnified/Bundle/SearchBundleFindologic/QueryBuilder/NewQueryBuilder.php]: The name of the parameter is `usergrouphash` and the method should be `addUserGroup`.

File: FinSearchUnified/composer.json [hunk 2]
```diff
@@ -21,7 +21,8 @@
   },
   "require": {
     "php": ">=5.6.4",
-    "findologic/libflexport": "~1.2"
+    "findologic/libflexport": "~1.2",
+    "findologic/findologic-api": "~1.2"
```
Comment (TheKeymaster) [inline FinSearchUnified/composer.json]: You should be able to get `1.3` now.

File: FinSearchUnified/Subscriber/RegisterComponents.php [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 namespace FinSearchUnified\Subscriber;
 
+use Composer\Autoload\ClassLoader;
```
Comment (TheKeymaster) [inline FinSearchUnified/Subscriber/RegisterComponents.php]: This import is not needed.

Comment (zaifastafa) [reply FinSearchUnified/Subscriber/RegisterComponents.php]: @TheKeymaster why so?

**Pull request merged**